static/locales/en-US.yaml

```
@@ -1086,6 +1086,7 @@ Tooltips:
        The channel ID entered must be a complete match and is case sensitive.
     Hide Subscriptions Live: 'This setting is overridden by the app-wide "{appWideSetting}" setting, in the "{subsection}" section of the "{settingsSection}"'
     Hide Videos and Playlists Containing Text: Enter a word, word fragment, or phrase (case insensitive) to hide all videos & playlists whose original titles contain it throughout all of FreeTube, excluding only History, Your Playlists, and videos inside of playlists.
+    Hide Videos on Watch: 'This option hides watched videos, lives and shorts from the Subscriptions and Channels Pages. It does not affect the Home tab on the Channels page"'
```

I reread this again and thought it wasnt very clear

```suggestion
    Hide Videos on Watch: 'Hides watched videos from the Videos, Shorts, and Live tabs on the Subscription and Channel pages. This does not affect the Home tab on Channel pages'
```

---

This one needs to be commited again